applications/crossbar/src/modules/cb_search.erl

```diff
@@ -62,6 +66,14 @@ allowed_methods() ->
 -spec resource_exists() -> 'true'.
 resource_exists() -> 'true'.
 
+-spec authenticate(cb_context:context()) -> 'true'.
+authenticate(Context) ->
```

Comment:
This (and the binding) is not required.  cb_token_auth will take care of this.  You should only need to bind for authorization since there is no account id.